components/switch/index.less

```diff
@@ -38,37 +37,24 @@
                 text-align: center;
                 position: relative;
                 z-index: 2;
-                transition: all 0.2s ease-in 0.0s;
+                color: #005d8e;
```

Comment:
We should add a color spec for this as we are going to be using it for the `done` button too.